Library/ViewModels/SettingsViewModel.swift

```diff
@@ -453,6 +489,7 @@ private enum Notification {
   case mobilePostLikes
   case mobileUpdates
   case postLikes
+  case creatorTips
```

Comment:
Could you move `case creatorTips` up so we keep the enumeration in alphabetical order?